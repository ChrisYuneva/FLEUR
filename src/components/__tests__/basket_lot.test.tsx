import React from 'react';
import { mount } from 'enzyme';
import { BasketLot } from '../index';
import img from '../../asset';
import { describe, it, expect } from '@jest/globals';

describe('<BasketLot />', () => {
    const name = 'name';
    const price = 'price';

    it('Отрисовывается без ошибок', () => {
        const wrapper = mount(
            <BasketLot img={img} name={name} price={price} />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
