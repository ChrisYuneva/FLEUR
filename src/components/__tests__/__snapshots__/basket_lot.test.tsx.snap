// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasketLot /> Отрисовывается без ошибок 1`] = `
<BasketLot
  img={
    Object {
      "accessories_1": "FILE_MOCK",
      "accessories_10": "FILE_MOCK",
      "accessories_11": "FILE_MOCK",
      "accessories_12": "FILE_MOCK",
      "accessories_13": "FILE_MOCK",
      "accessories_14": "FILE_MOCK",
      "accessories_15": "FILE_MOCK",
      "accessories_2": "FILE_MOCK",
      "accessories_3": "FILE_MOCK",
      "accessories_4": "FILE_MOCK",
      "accessories_5": "FILE_MOCK",
      "accessories_6": "FILE_MOCK",
      "accessories_7": "FILE_MOCK",
      "accessories_8": "FILE_MOCK",
      "accessories_9": "FILE_MOCK",
      "background2": "FILE_MOCK",
      "backgroung": "FILE_MOCK",
      "dress1": "FILE_MOCK",
      "dress10": "FILE_MOCK",
      "dress11": "FILE_MOCK",
      "dress12": "FILE_MOCK",
      "dress13": "FILE_MOCK",
      "dress14": "FILE_MOCK",
      "dress2": "FILE_MOCK",
      "dress3": "FILE_MOCK",
      "dress4": "FILE_MOCK",
      "dress5": "FILE_MOCK",
      "dress6": "FILE_MOCK",
      "dress7": "FILE_MOCK",
      "dress8": "FILE_MOCK",
      "dress9": "FILE_MOCK",
      "hoodies_m_1": "FILE_MOCK",
      "hoodies_m_10": "FILE_MOCK",
      "hoodies_m_11": "FILE_MOCK",
      "hoodies_m_12": "FILE_MOCK",
      "hoodies_m_2": "FILE_MOCK",
      "hoodies_m_3": "FILE_MOCK",
      "hoodies_m_4": "FILE_MOCK",
      "hoodies_m_5": "FILE_MOCK",
      "hoodies_m_6": "FILE_MOCK",
      "hoodies_m_7": "FILE_MOCK",
      "hoodies_m_8": "FILE_MOCK",
      "hoodies_m_9": "FILE_MOCK",
      "hoodies_w_1": "FILE_MOCK",
      "hoodies_w_10": "FILE_MOCK",
      "hoodies_w_11": "FILE_MOCK",
      "hoodies_w_12": "FILE_MOCK",
      "hoodies_w_13": "FILE_MOCK",
      "hoodies_w_14": "FILE_MOCK",
      "hoodies_w_2": "FILE_MOCK",
      "hoodies_w_3": "FILE_MOCK",
      "hoodies_w_4": "FILE_MOCK",
      "hoodies_w_5": "FILE_MOCK",
      "hoodies_w_6": "FILE_MOCK",
      "hoodies_w_7": "FILE_MOCK",
      "hoodies_w_8": "FILE_MOCK",
      "hoodies_w_9": "FILE_MOCK",
      "icon": "FILE_MOCK",
      "icon2": "FILE_MOCK",
      "icon3": "FILE_MOCK",
      "icon4": "FILE_MOCK",
      "jackets_m_1": "FILE_MOCK",
      "jackets_m_10": "FILE_MOCK",
      "jackets_m_11": "FILE_MOCK",
      "jackets_m_12": "FILE_MOCK",
      "jackets_m_13": "FILE_MOCK",
      "jackets_m_2": "FILE_MOCK",
      "jackets_m_3": "FILE_MOCK",
      "jackets_m_4": "FILE_MOCK",
      "jackets_m_5": "FILE_MOCK",
      "jackets_m_6": "FILE_MOCK",
      "jackets_m_7": "FILE_MOCK",
      "jackets_m_8": "FILE_MOCK",
      "jackets_m_9": "FILE_MOCK",
      "jackets_w_1": "FILE_MOCK",
      "jackets_w_10": "FILE_MOCK",
      "jackets_w_11": "FILE_MOCK",
      "jackets_w_2": "FILE_MOCK",
      "jackets_w_3": "FILE_MOCK",
      "jackets_w_4": "FILE_MOCK",
      "jackets_w_5": "FILE_MOCK",
      "jackets_w_6": "FILE_MOCK",
      "jackets_w_7": "FILE_MOCK",
      "jackets_w_8": "FILE_MOCK",
      "jackets_w_9": "FILE_MOCK",
      "jeans_m_1": "FILE_MOCK",
      "jeans_m_10": "FILE_MOCK",
      "jeans_m_11": "FILE_MOCK",
      "jeans_m_12": "FILE_MOCK",
      "jeans_m_13": "FILE_MOCK",
      "jeans_m_2": "FILE_MOCK",
      "jeans_m_3": "FILE_MOCK",
      "jeans_m_4": "FILE_MOCK",
      "jeans_m_5": "FILE_MOCK",
      "jeans_m_6": "FILE_MOCK",
      "jeans_m_7": "FILE_MOCK",
      "jeans_m_8": "FILE_MOCK",
      "jeans_m_9": "FILE_MOCK",
      "jeans_w_1": "FILE_MOCK",
      "jeans_w_10": "FILE_MOCK",
      "jeans_w_11": "FILE_MOCK",
      "jeans_w_12": "FILE_MOCK",
      "jeans_w_13": "FILE_MOCK",
      "jeans_w_2": "FILE_MOCK",
      "jeans_w_3": "FILE_MOCK",
      "jeans_w_4": "FILE_MOCK",
      "jeans_w_5": "FILE_MOCK",
      "jeans_w_6": "FILE_MOCK",
      "jeans_w_7": "FILE_MOCK",
      "jeans_w_8": "FILE_MOCK",
      "jeans_w_9": "FILE_MOCK",
      "shirt_m_1": "FILE_MOCK",
      "shirt_m_10": "FILE_MOCK",
      "shirt_m_11": "FILE_MOCK",
      "shirt_m_12": "FILE_MOCK",
      "shirt_m_2": "FILE_MOCK",
      "shirt_m_3": "FILE_MOCK",
      "shirt_m_4": "FILE_MOCK",
      "shirt_m_5": "FILE_MOCK",
      "shirt_m_6": "FILE_MOCK",
      "shirt_m_7": "FILE_MOCK",
      "shirt_m_8": "FILE_MOCK",
      "shirt_m_9": "FILE_MOCK",
      "shirt_w_1": "FILE_MOCK",
      "shirt_w_10": "FILE_MOCK",
      "shirt_w_11": "FILE_MOCK",
      "shirt_w_12": "FILE_MOCK",
      "shirt_w_2": "FILE_MOCK",
      "shirt_w_3": "FILE_MOCK",
      "shirt_w_4": "FILE_MOCK",
      "shirt_w_5": "FILE_MOCK",
      "shirt_w_6": "FILE_MOCK",
      "shirt_w_7": "FILE_MOCK",
      "shirt_w_8": "FILE_MOCK",
      "shirt_w_9": "FILE_MOCK",
      "skirts_1": "FILE_MOCK",
      "skirts_10": "FILE_MOCK",
      "skirts_11": "FILE_MOCK",
      "skirts_12": "FILE_MOCK",
      "skirts_2": "FILE_MOCK",
      "skirts_3": "FILE_MOCK",
      "skirts_4": "FILE_MOCK",
      "skirts_5": "FILE_MOCK",
      "skirts_6": "FILE_MOCK",
      "skirts_7": "FILE_MOCK",
      "skirts_8": "FILE_MOCK",
      "skirts_9": "FILE_MOCK",
      "tshirt_m_1": "FILE_MOCK",
      "tshirt_m_10": "FILE_MOCK",
      "tshirt_m_2": "FILE_MOCK",
      "tshirt_m_3": "FILE_MOCK",
      "tshirt_m_4": "FILE_MOCK",
      "tshirt_m_5": "FILE_MOCK",
      "tshirt_m_6": "FILE_MOCK",
      "tshirt_m_7": "FILE_MOCK",
      "tshirt_m_8": "FILE_MOCK",
      "tshirt_m_9": "FILE_MOCK",
      "tshirt_w_1": "FILE_MOCK",
      "tshirt_w_10": "FILE_MOCK",
      "tshirt_w_11": "FILE_MOCK",
      "tshirt_w_12": "FILE_MOCK",
      "tshirt_w_13": "FILE_MOCK",
      "tshirt_w_2": "FILE_MOCK",
      "tshirt_w_3": "FILE_MOCK",
      "tshirt_w_4": "FILE_MOCK",
      "tshirt_w_5": "FILE_MOCK",
      "tshirt_w_6": "FILE_MOCK",
      "tshirt_w_7": "FILE_MOCK",
      "tshirt_w_8": "FILE_MOCK",
      "tshirt_w_9": "FILE_MOCK",
    }
  }
  name="name"
  price="price"
>
  <div
    className="wrap"
  >
    <div
      className="showlittle"
    >
      <div
        className="image"
      >
        <img
          src={
            Object {
              "accessories_1": "FILE_MOCK",
              "accessories_10": "FILE_MOCK",
              "accessories_11": "FILE_MOCK",
              "accessories_12": "FILE_MOCK",
              "accessories_13": "FILE_MOCK",
              "accessories_14": "FILE_MOCK",
              "accessories_15": "FILE_MOCK",
              "accessories_2": "FILE_MOCK",
              "accessories_3": "FILE_MOCK",
              "accessories_4": "FILE_MOCK",
              "accessories_5": "FILE_MOCK",
              "accessories_6": "FILE_MOCK",
              "accessories_7": "FILE_MOCK",
              "accessories_8": "FILE_MOCK",
              "accessories_9": "FILE_MOCK",
              "background2": "FILE_MOCK",
              "backgroung": "FILE_MOCK",
              "dress1": "FILE_MOCK",
              "dress10": "FILE_MOCK",
              "dress11": "FILE_MOCK",
              "dress12": "FILE_MOCK",
              "dress13": "FILE_MOCK",
              "dress14": "FILE_MOCK",
              "dress2": "FILE_MOCK",
              "dress3": "FILE_MOCK",
              "dress4": "FILE_MOCK",
              "dress5": "FILE_MOCK",
              "dress6": "FILE_MOCK",
              "dress7": "FILE_MOCK",
              "dress8": "FILE_MOCK",
              "dress9": "FILE_MOCK",
              "hoodies_m_1": "FILE_MOCK",
              "hoodies_m_10": "FILE_MOCK",
              "hoodies_m_11": "FILE_MOCK",
              "hoodies_m_12": "FILE_MOCK",
              "hoodies_m_2": "FILE_MOCK",
              "hoodies_m_3": "FILE_MOCK",
              "hoodies_m_4": "FILE_MOCK",
              "hoodies_m_5": "FILE_MOCK",
              "hoodies_m_6": "FILE_MOCK",
              "hoodies_m_7": "FILE_MOCK",
              "hoodies_m_8": "FILE_MOCK",
              "hoodies_m_9": "FILE_MOCK",
              "hoodies_w_1": "FILE_MOCK",
              "hoodies_w_10": "FILE_MOCK",
              "hoodies_w_11": "FILE_MOCK",
              "hoodies_w_12": "FILE_MOCK",
              "hoodies_w_13": "FILE_MOCK",
              "hoodies_w_14": "FILE_MOCK",
              "hoodies_w_2": "FILE_MOCK",
              "hoodies_w_3": "FILE_MOCK",
              "hoodies_w_4": "FILE_MOCK",
              "hoodies_w_5": "FILE_MOCK",
              "hoodies_w_6": "FILE_MOCK",
              "hoodies_w_7": "FILE_MOCK",
              "hoodies_w_8": "FILE_MOCK",
              "hoodies_w_9": "FILE_MOCK",
              "icon": "FILE_MOCK",
              "icon2": "FILE_MOCK",
              "icon3": "FILE_MOCK",
              "icon4": "FILE_MOCK",
              "jackets_m_1": "FILE_MOCK",
              "jackets_m_10": "FILE_MOCK",
              "jackets_m_11": "FILE_MOCK",
              "jackets_m_12": "FILE_MOCK",
              "jackets_m_13": "FILE_MOCK",
              "jackets_m_2": "FILE_MOCK",
              "jackets_m_3": "FILE_MOCK",
              "jackets_m_4": "FILE_MOCK",
              "jackets_m_5": "FILE_MOCK",
              "jackets_m_6": "FILE_MOCK",
              "jackets_m_7": "FILE_MOCK",
              "jackets_m_8": "FILE_MOCK",
              "jackets_m_9": "FILE_MOCK",
              "jackets_w_1": "FILE_MOCK",
              "jackets_w_10": "FILE_MOCK",
              "jackets_w_11": "FILE_MOCK",
              "jackets_w_2": "FILE_MOCK",
              "jackets_w_3": "FILE_MOCK",
              "jackets_w_4": "FILE_MOCK",
              "jackets_w_5": "FILE_MOCK",
              "jackets_w_6": "FILE_MOCK",
              "jackets_w_7": "FILE_MOCK",
              "jackets_w_8": "FILE_MOCK",
              "jackets_w_9": "FILE_MOCK",
              "jeans_m_1": "FILE_MOCK",
              "jeans_m_10": "FILE_MOCK",
              "jeans_m_11": "FILE_MOCK",
              "jeans_m_12": "FILE_MOCK",
              "jeans_m_13": "FILE_MOCK",
              "jeans_m_2": "FILE_MOCK",
              "jeans_m_3": "FILE_MOCK",
              "jeans_m_4": "FILE_MOCK",
              "jeans_m_5": "FILE_MOCK",
              "jeans_m_6": "FILE_MOCK",
              "jeans_m_7": "FILE_MOCK",
              "jeans_m_8": "FILE_MOCK",
              "jeans_m_9": "FILE_MOCK",
              "jeans_w_1": "FILE_MOCK",
              "jeans_w_10": "FILE_MOCK",
              "jeans_w_11": "FILE_MOCK",
              "jeans_w_12": "FILE_MOCK",
              "jeans_w_13": "FILE_MOCK",
              "jeans_w_2": "FILE_MOCK",
              "jeans_w_3": "FILE_MOCK",
              "jeans_w_4": "FILE_MOCK",
              "jeans_w_5": "FILE_MOCK",
              "jeans_w_6": "FILE_MOCK",
              "jeans_w_7": "FILE_MOCK",
              "jeans_w_8": "FILE_MOCK",
              "jeans_w_9": "FILE_MOCK",
              "shirt_m_1": "FILE_MOCK",
              "shirt_m_10": "FILE_MOCK",
              "shirt_m_11": "FILE_MOCK",
              "shirt_m_12": "FILE_MOCK",
              "shirt_m_2": "FILE_MOCK",
              "shirt_m_3": "FILE_MOCK",
              "shirt_m_4": "FILE_MOCK",
              "shirt_m_5": "FILE_MOCK",
              "shirt_m_6": "FILE_MOCK",
              "shirt_m_7": "FILE_MOCK",
              "shirt_m_8": "FILE_MOCK",
              "shirt_m_9": "FILE_MOCK",
              "shirt_w_1": "FILE_MOCK",
              "shirt_w_10": "FILE_MOCK",
              "shirt_w_11": "FILE_MOCK",
              "shirt_w_12": "FILE_MOCK",
              "shirt_w_2": "FILE_MOCK",
              "shirt_w_3": "FILE_MOCK",
              "shirt_w_4": "FILE_MOCK",
              "shirt_w_5": "FILE_MOCK",
              "shirt_w_6": "FILE_MOCK",
              "shirt_w_7": "FILE_MOCK",
              "shirt_w_8": "FILE_MOCK",
              "shirt_w_9": "FILE_MOCK",
              "skirts_1": "FILE_MOCK",
              "skirts_10": "FILE_MOCK",
              "skirts_11": "FILE_MOCK",
              "skirts_12": "FILE_MOCK",
              "skirts_2": "FILE_MOCK",
              "skirts_3": "FILE_MOCK",
              "skirts_4": "FILE_MOCK",
              "skirts_5": "FILE_MOCK",
              "skirts_6": "FILE_MOCK",
              "skirts_7": "FILE_MOCK",
              "skirts_8": "FILE_MOCK",
              "skirts_9": "FILE_MOCK",
              "tshirt_m_1": "FILE_MOCK",
              "tshirt_m_10": "FILE_MOCK",
              "tshirt_m_2": "FILE_MOCK",
              "tshirt_m_3": "FILE_MOCK",
              "tshirt_m_4": "FILE_MOCK",
              "tshirt_m_5": "FILE_MOCK",
              "tshirt_m_6": "FILE_MOCK",
              "tshirt_m_7": "FILE_MOCK",
              "tshirt_m_8": "FILE_MOCK",
              "tshirt_m_9": "FILE_MOCK",
              "tshirt_w_1": "FILE_MOCK",
              "tshirt_w_10": "FILE_MOCK",
              "tshirt_w_11": "FILE_MOCK",
              "tshirt_w_12": "FILE_MOCK",
              "tshirt_w_13": "FILE_MOCK",
              "tshirt_w_2": "FILE_MOCK",
              "tshirt_w_3": "FILE_MOCK",
              "tshirt_w_4": "FILE_MOCK",
              "tshirt_w_5": "FILE_MOCK",
              "tshirt_w_6": "FILE_MOCK",
              "tshirt_w_7": "FILE_MOCK",
              "tshirt_w_8": "FILE_MOCK",
              "tshirt_w_9": "FILE_MOCK",
            }
          }
        />
        <span>
          name
           
          <br />
           
          price
        </span>
      </div>
    </div>
  </div>
</BasketLot>
`;
